src/routes/docs/products/sites/quick-start/tanstack-start/+page.markdoc

```diff
@@ -0,0 +1,52 @@
+---
+layout: article
+title: Deploy a Tanstack Start app to Appwrite Sites
+description: Learn how to setup and deploy Tanstack Start apps on Appwrite Sites.
+---
+
+{% section #step-1 step=1 title="Create Tanstack Start app" %}
+
+First, you must either create a Tanstack Start app or setup the [Tanstack Start starter template](https://github.com/appwrite/templates-for-sites).
+
+Open your terminal, and run the following command.
+
+```bash
+npm create @tanstack/start@latest
+```
+
+Push this project to a [GitHub repo](https://github.com/new).
```

Comment:
```suggestion
Push this project to a [GitHub repository](https://github.com/new).
```

Reply:
This is consistent as such across all quickstarts, I'll update everywhere